RELEASING.md

```diff
@@ -0,0 +1,66 @@
+# Release Process
+
+This doc covers the release process for the functions in the
+kpt-functions-catalog repo.
+
+1. Checking the [CI status](https://github.com/kptdev/krm-functions-catalog/actions/workflows/ci.yaml) of the master branch. 
+   If the CI is failing on the master, we need to fix it before doing a release.
+1. Go to the [releases pages] in your browser.
+1. Click `Draft a new release` to create a new release for a function. The tag
+   version format should be `functions/{language}/{function-name}/{semver}`. e.g.
+   `functions/go/set-namespace/v1.2.3` and `functions/ts/kubeval/v2.3.4`. The release name should be
```

Comment:
Do we keep the theoretical possibility to support multiple languages? I think we should maybe in the future we will have a maintainer for other ecosystems than Go.